patches.luci/0030-swconfig-support-port-aliases-cpu-wan-lan.patch

```diff
@@ -38,32 +38,7 @@ Index: luci/modules/luci-mod-network/htdocs/luci-static/resources/view/network/s
  		if (m && m[1] == this.option)
  			return m[2] || 'u';
  	}
-@@ -295,16 +298,22 @@ return L.view.extend({
- 
- 			o.write = function(section_id, value) {
- 				var topology = this.section.topology,
-+				    port,
- 				    values = [];
- 
- 				for (var i = 0; i < port_opts.length; i++) {
- 					var tagging = port_opts[i].formvalue(section_id),
- 					    portspec = Array.isArray(topology.ports) ? topology.ports[i] : null;
- 
-+					if (port_opts[i].role)
-+						port = port_opts[i].role;
-+					else
-+						port = port_opts[i].option;
-+
- 					if (tagging == 't')
--						values.push(port_opts[i].option + tagging);
-+						values.push(port + tagging);
- 					else if (tagging == 'u')
--						values.push(port_opts[i].option);
-+						values.push(port);
- 
- 					if (portspec && portspec.device) {
- 						var old_tag = port_opts[i].cfgvalue(section_id),
-@@ -347,6 +356,7 @@ return L.view.extend({
+@@ -355,6 +358,7 @@ return view.extend({
```

Comment:
This commit effectively removes the port aliases feature from the GUI. The switch config page for the older APs will likely not function correctly from the GUI.